opentech/apply/funds/models/submissions.py

```diff
@@ -566,6 +566,10 @@ def save(self, *args, update_fields=list(), **kwargs):
                 form_data=self.form_data,
                 author=self.user,
             )
+
+            if hasattr(self, 'previous'):
```

Comment:
self.previous is not set at this stage, tips to get it working?

Reply:
@frjo This code runs when an application moves from one stage to another.

https://github.com/OpenTechFund/opentech.fund/blob/322c41732872b9dc1e208273ff970fdcdb8066de/opentech/apply/funds/models/submissions.py#L443-L464

We can also send notifications to partners by updating code [here](https://github.com/OpenTechFund/opentech.fund/blob/322c41732872b9dc1e208273ff970fdcdb8066de/opentech/apply/funds/models/submissions.py#L689).